### ISSUE TITLE
1-quality.txt breaks on clean systems

### ISSUE THREAD

**Issue by bocajnotnef** [ged-lab/khmer-protocols#154]
https://github.com/ged-lab/khmer-protocols/blob/master/mrnaseq/1-quality.txt#L48

needs || true appended, for situations where this is a clean environment. Otherwise the entire test breaks.

**Comment by bocajnotnef** [comment]
Also, trinity installation fails:

from `3-big-assembly.out`:

```+ set -e
+ rm -fr /root/trinity.tar.gz '/root/bowtie*' '/root/samtools*'
+ rm -fr /mnt/work/trinity_out_dir
+ rm -f '/mnt/work/*.1' '/mnt/work/*.2'
++ date
+ echo 3-big-assembly compileTrinity Sat Mar 7 22:28:25 CST 2015
+ cd /root
+ curl -L 'http://sourceforge.net/projects/trinityrnaseq/files/latest/download?source=files'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M  0   407    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
^M  0   321    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
^M100   125  100   125    0     0    364      0 --:--:-- --:--:-- --:--:--   364
+ tar xzf trinity.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
``

**Comment by bocajnotnef** [comment]
Fix to be backported from master:

```
  wget https://github.com/trinityrnaseq/trinityrnaseq/archive/v2.0.4.tar.gz \
     -O trinity.tar.gz
   tar xzf trinity.tar.gz
   cd trinityrnaseq*/
   make |& tee trinity-build.log
```

**Comment by bocajnotnef** [comment]
Closed #156
